docs/index.rst

```diff
@@ -1,255 +1,50 @@
-    
-    @app.route('/photo/<id>')
-    def show(id):
-        photo = Photo.load(id)
-        if photo is None:
-            abort(404)
-        url = photos.url(photo.filename)
-        return render_template('show.html', url=url, photo=photo)
-
-If you have a "default location" for storing uploads - for example, if your
-app has an "instance" directory and uploads should be saved to
-the instance directory's ``uploads`` folder - you can pass a ``default_dest``
-callable to the set constructor. It takes the application as its argument.
-For example::
-
-    media = UploadSet('media', default_dest=lambda app: app.instance_path)
-
-This won't prevent a different destination from being set in the config,
-though. It's just to save your users a little configuration time.
-
-
-App Configuration
-=================
-An `UploadSet`'s configuration is stored on an app. That way, you can have
-upload sets being used by multiple apps at once. You use the
-`configure_uploads` function to load the configuration for the `UploadSet`s.
-You pass in the app and all of the `UploadSet`s you want configured. Calling
-`configure_uploads` more than once is safe. ::
-
-    configure_uploads(app, (photos, media))
-
-If your app has a factory function, that is a good place to call this
-function.
-
-
-File Upload Forms
-=================
-To actually upload the files, you need to properly set up your form. A form
-that uploads files needs to have its method set to POST and its enctype
-set to ``multipart/form-data``. If it's set to GET, it won't work at all, and
-if you don't set the enctype, only the filename will be transferred.
-
-The field itself should be an ``<input type=file>``.
-
-.. code-block:: html+jinja
-
-    <form method=POST enctype=multipart/form-data action="{{ url_for('upload') }}">
-        ...
-        <input type=file name=photo>
-        ...
-    </form>
-
-
-API Documentation
-=================
-This documentation is generated directly from the source code.
-
-
-Upload Sets
------------
-.. autoclass:: UploadSet
-   :members:
-
-.. autoclass:: UploadConfiguration
-
-
-Application Setup
------------------
-.. autofunction:: configure_uploads
-
-
-Extension Constants
--------------------
-These are some default sets of extensions you can pass to the `UploadSet`
-constructor.
-
-.. autoclass:: AllExcept
-
-.. autodata:: DEFAULTS
-
-.. autodata:: ALL
-
-.. autodata:: TEXT
-
-.. autodata:: IMAGES
-
-.. autodata:: AUDIO
-
-.. autodata:: DOCUMENTS
-
-.. autodata:: DATA
-
-.. autodata:: SCRIPTS
-
-.. autodata:: ARCHIVES
 
-.. autodata:: EXECUTABLES
+Notes on this package
+---------------------
 
+This is an independently maintained version of `Flask-Uploads`
+including four years of unreleased changes, at least not released to PyPI.
 
-Testing Utilities
------------------
-.. autoclass:: TestingFileStorage
+Noteworthy is the fix for the `Werkzeug` API change.
+Please see the migration guide from `flask-Uploads`
```

Comment:
```suggestion
Please see the migration guide from `Flask-Uploads`
```

As above... we need a link to the migration guide. As this is the `index.rst` you can actually use rst syntax to link to documents.

see https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#explicit-links